js/checkout.js

```diff
@@ -0,0 +1,242 @@
+            let check_script = () => {
+
+                let is_loaded = true
+                document.querySelectorAll('script.controlled-by-bnomics').forEach(script => {
+                    if (!script.hasAttribute('bnomics-loaded') && !script.hasAttribute('bnomics-error')) {
+                        is_loaded = false
+                    }
+                })
+
+                if (is_loaded) resolve()
+                else setTimeout(check_script, 1000)
+            }
+
+            setTimeout(check_script, 1000)
+        })
+
+    }
+
+    create_bindings() {
+        this._spinner_wrapper = this.container.querySelector('.bnomics-spinner-wrapper')
+
+        this._order_expired_wrapper = this.container.querySelector('.bnomics-order-expired-wrapper')
+        this._order_panel = this.container.querySelector('.bnomics-order-panel')
+
+        this._amount_text = this.container.querySelector('.bnomics-amount-text')
+        this._copy_amount_text = this.container.querySelector('.bnomics-copy-amount-text')
+        this._amount_input = this.container.querySelector('#bnomics-amount-input')
+        this._amount_copy = this.container.querySelector('#bnomics-amount-copy')
+
+        this._address_text = this.container.querySelector('.bnomics-address-text')
+        this._copy_address_text = this.container.querySelector('.bnomics-copy-address-text')
+        this._address_input = this.container.querySelector('#bnomics-address-input')
+        this._address_copy = this.container.querySelector('.bnomics-address-copy')
+
+        this._progress_bar = this.container.querySelector('.bnomics-progress-bar')
+        this._time_left = this.container.querySelector('.bnomics-time-left')
+
+        this._try_again = this.container.querySelector('#bnomics-try-again')
+        this._qr_code = this.container.querySelector('#bnomics-qr-code')
+        
+        // Hide Panels
+        this._order_expired_wrapper.style.display = 'none'
+        this._order_panel.style.display = 'none'
+        this._copy_amount_text.style.display = 'none'
+        this._copy_address_text.style.display = 'none'
+
+        // Click Bindings
+
+        //Copy bitcoin address to clipboard
+        this._address_input.addEventListener('click', () => this.copy_to_clipboard("bnomics-address-copy"))
+        
+        //Copy bitcoin amount to clipboard
+        this._amount_input.addEventListener('click', () => this.copy_to_clipboard("bnomics-amount-copy"))
+
+        //Reload the page if user clicks try again after the order expires
+        this._try_again.addEventListener('click', () => location.reload())
+
+        // Load data attributes
+        let data_container = this.container.querySelector(".blockonomics-data")
+        Object.keys(data_container.dataset).forEach(key => {
+            this.data[key] = data_container.dataset[key]
+        })
+        this.data.time_period = isNaN(this.data.time_period) ? 10 : Number(this.data.time_period)
```

Comment:
When collecting the time_period in php we are already setting the default value to 10: https://github.com/blockonomics/woocommerce-plugin/pull/276/files#diff-8d81ebce00651fad5f0a421bc7d60d514e8ca6bd12cc9fc434ab5e46dab827ecR33